Firmware/ultralcd.cpp

```diff
@@ -7284,6 +7280,7 @@ void menu_lcd_lcdupdate_func(void)
 	{
 		if (lcd_draw_update) {
 			lcd_timeoutToStatus.start();
+            lcd_frame_start();
```

Comment:
@leptun I wonder, do we want to call this function also when the knob is rotated? I'm thinking if we can then remove the call to `lcd_frame_start()` in `lcd_display_message_fullscreen_nonBlocking_P`

`lcd_knob_update` is called even when LCD updates are disabled. I wonder if that will suffice to cover all fullscreen messages.

Reply:
Idk, it feels weird tying the start of a frame with the knob update. Need to think more about this.

Reply:
It seems to work as it is so I won't complain :)